portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -177,12 +177,48 @@ definition {
 	}
 
 	@description = "Verify that a Table can be created in the Body Text"
-	@ignore = "true"
 	@priority = "4"
 	test CanSupportTables {
 		property portal.acceptance = "false";
```

Comment:
Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

Reply:
@natocesarrego done!